core/src/main/scripts/migrate_db.py

```diff
@@ -0,0 +1,204 @@
+#! /usr/bin/env python
+
```

Comment:
I think this script is a great idea. A few comments though:
- I still miss the part where the web application backend checks if it is running on the correct version of the DB. My idea is that, at startup, the backend should check if the DB has been migrated to the version that it is expecting it to be in. And if the migration did not yet occur, ask the user to run DB migration (this script) first. See also [my comment on 796] (https://github.com/cBioPortal/cbioportal/issues/796#issuecomment-177917780) here. 
@jjgao : any thoughts? Or will this be part of another PR?